prerendering.bs

```diff
@@ -734,6 +734,9 @@ APIs that require [=transient activation=] or [=sticky activation=]:
 - {{Window/open(url, target, features)|window.open()}} [[HTML]]
 - The prompt generated by the {{Window/beforeunload}} event [[HTML]]
 - {{Element/requestFullscreen()|element.requestFullscreen()}} [[FULLSCREEN]]
+  - {{Keyboard/lock()|navigator.keyboard.lock()}} [[KEYBOARD-LOCK]], which requires fullscreen
+
+    <div class="note">[[KEYBOARD-LOCK]] allows browsing contexts to enable keyboard lock easily but it has no effect except in fullscreen, which requires a user gesture. If that changes then this should be revisited.</div>
```

Comment:
should -> needs to be to avoid using special keywords

Reply:
Bah, I ran the build the first time but thought I could get away with amending the note without messing this up. Fixed.